hammer/config/defaults.yml

```diff
@@ -60,6 +60,12 @@ vlsi.technology:
   bump_block_cut_layer: null # Top cut/via layer for blockage under bumps. (Optional[str])
   # Only used if using vlsi.inputs.bumps
   # TODO: remove this after stackup supports vias ucb-bar/hammer#354
+  
+  tap_cell_interval: 10.0 # Spacing between columns of tap cells (float)
```

Comment:
Is there any particular reason at this point we're using a mix of Decimal and float types for keys?

Reply:
This is not Decimal because it's not processed anywhere in Hammer - it's directly read by tool plugins and so remains as a float.

Reply:
But, I could see how mixing this in the defaults file could create some confusion. Pydantic would remove this.